bloom/generators/debian/generator.py

```diff
@@ -430,18 +459,17 @@ def generate_substitutions_from_package(
     summarize_dependency_mapping(data, depends, build_depends, resolved_deps)
     # Copyright
     licenses = []
-    separator = '\n' + '=' * 80 + '\n\n'
     for l in package.licenses:
         if hasattr(l, 'file') and l.file is not None:
             license_file = os.path.join(os.path.dirname(package.filename), l.file)
             if not os.path.exists(license_file):
                 error("License file '{}' is not found.".
                       format(license_file), exit=True)
-            license_text = open(license_file, 'r').read()
-            if not license_text.endswith('\n'):
-                license_text += '\n'
-            licenses.append(license_text)
-    data['Copyright'] = separator.join(licenses)
+            license_text = open(license_file, 'r').read().rstrip()
+            licenses.append((str(l), format_multiline(license_text)))
+        else:
+            licenses.append((str(l), ''))
```

Comment:
From: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#license-field

> If there are no remaining lines, then all of the short names or short names followed by license exceptions in the synopsis must be described in stand-alone License paragraphs. Otherwise, this field should either include the full text of the license(s) or include a pointer to the license file under /usr/share/common-licenses. This field should include all text needed in order to fulfill both Debian Policy's requirement for including a copy of the software's distribution license (12.5), and any license requirements to include warranty disclaimers or other notices with the binary package. 

The way I interpret the above paragraph, our copyright file will not be valid without at least one copy of the full license text which would result in malformed debian/copyright files when package.xml files have licenses without file attributes.

Reply:
This was an oversight on my part, thanks for catching it. I've added some placeholder text referring the consumer to the repository itself to find the full text.